### PR TITLE
add C++ versions of the conductivities

### PR DESCRIPTION
**Pull request by zingale**
As part of this change, the stellar conductivity was updated to the constants from the fundamental_constants_module (for both Fortran and C++)

### CODE REVIEW

File: conductivity/constant/Make.package [hunk 1]
```diff
@@ -1 +1,3 @@
 F90EXE_sources += constant_conductivity.F90
+CEXE_headers += actual_conductivity.H
+CEXE_headers += actual_conductivity_data.H
```
Comment (maxpkatz) [inline conductivity/constant/Make.package]: Does this file exist and it wasn't checked in? Or it it just an incorrect addition to the Make.package?

File: conductivity/constant_opacity/Make.package [hunk 2]
```diff
@@ -1,2 +1,3 @@
 F90EXE_sources += constant_conductive_opacity.F90
-
+CEXE_headers += actual_conductivity.H
+CEXE_headers += actual_conductivity_data.H
```
Comment (maxpkatz) [inline conductivity/constant_opacity/Make.package]: Same comment

File: conductivity/powerlaw/Make.package [hunk 3]
```diff
@@ -1,2 +1,3 @@
 F90EXE_sources += powerlaw_conductivity.F90
-
+CEXE_headers += conductivity.H
+CEXE_headers += conductivity_data.H
```
Comment (maxpkatz) [inline conductivity/powerlaw/Make.package]: Same comment

File: conductivity/stellar/actual_conductivity.H [hunk 4]
```diff
@@ -0,0 +1,347 @@
...
+  } else {
+    if (xz > zbound) {
+      orad = oiben1;
+    } else {
+      orad = oiben1*(xz/zbound) + oiben2*((zbound-xz)/zbound);
+    }
+  }
+
+  // add in the compton scattering opacity, weaver et al. apj 1978 225 1021
+  Real th = std::min(511.0_rt, state.T * 8.617e-8_rt);
```
Comment (maxpkatz) [inline conductivity/stellar/actual_conductivity.H]: amrex::min

File: interfaces/conductivity.H [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+using namespace amrex;
+
+AMREX_FORCE_INLINE
+void conductivity_init() {
+  actual_conductivity_init();
+}
+
+
+AMREX_FORCE_INLINE
```
Comment (maxpkatz) [inline interfaces/conductivity.H]: Needs AMREX_GPU_HOST_DEVICE

File: conductivity/powerlaw/Make.package (hunk 3 above)
Comment (zingale) [reply conductivity/powerlaw/Make.package]: oh it did, but I removed the _data.H files.  I'll fix this

File: conductivity/powerlaw/Make.package [hunk 6]
```diff
@@ -1,2 +1,3 @@
 F90EXE_sources += powerlaw_conductivity.F90
+CEXE_headers += conductivity.H
```
Comment (maxpkatz) [inline conductivity/powerlaw/Make.package]: This needs to be actual_conductivity.H

**Pull request merged**